### PR TITLE
HPCC-19506 Unlock validation not showing errors

### PR DESCRIPTION
**Pull request by miguelvazq**
Allow the username and password to show when invalid and improve the
aesthetics of the lock dialog to mimic the login screen's appearance.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review.

**Review by GordonSmith — APPROVED** [review]
Looks good.

**Comment by richardkchapman** [comment]
@miguelvazq  What about the lint error that smoketest is reporting?

**Comment by miguelvazq** [comment]
@richardkchapman I'll fix the error.

File: esp/src/eclwatch/stub.js [hunk 1]
```diff
@@ -45,7 +45,6 @@ define([
             entities, Toaster) {
 
     var IDLE_TIMEOUT = cookie("ESPSessionTimeoutSeconds") * 1000;
-    var COUNTDOWN = 3 * 60;
```
Comment (miguelvazq) [inline esp/src/eclwatch/stub.js]: This was for previous countdown for logout.

**Comment by miguelvazq** [comment]
@GordonSmith Please re-review I found an issue with duplicate ID's in the dom causing conflicts so I added a destroyRecursive() to it.

**Review by GordonSmith — APPROVED** [review]
Looks good.

**Pull request merged**